imports/components/@enhancers/security-roles/index.js

```diff
@@ -0,0 +1,31 @@
+
+import { graphql } from "react-apollo";
+import { intersection } from "lodash";
+import gql from "graphql-tag";
+
+export const SECURITY_ROLES_QUERY = gql`
+  query GetSecurityRoles {
+    currentPerson{
+      roles {
+        name
+        id
+        groupId: entityId
+      }
+    }
+  }
+`;
+
+export const authorized = (data, roles) => {
+  const auth = !data.loading && intersection(data.roles.map((x) => x.name), roles).length > 0;
+  return auth;
+};
+
+export const canSee = (roles) => graphql(SECURITY_ROLES_QUERY, {
```

Comment:
Would we be wrapping this in a check for a `userId` when used on the page? Right now, this would fail if `roles` was passed as null.